PackageDownloadsExample/Models.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace PackageDownloadsExample
+{
+    /// <summary>
+    /// The response to a search query.
+    ///
+    /// See https://docs.microsoft.com/en-us/nuget/api/search-query-service-resource#response
+    /// </summary>
+    public class SearchResponse
```

Comment:
These models are necessary because of https://github.com/NuGet/Home/issues/9086

Reply:
You should fix it :D